ynab_sdk/api/models/responses/transactions.py

```diff
@@ -16,17 +16,26 @@ class Subtransaction:
     deleted: bool
 
     @staticmethod
-    def from_dict(obj: Any) -> 'Subtransaction':
+    def from_dict(obj: Any) -> "Subtransaction":
         assert isinstance(obj, dict)
         id = parsers.from_str(obj.get("id"))
         transaction_id = parsers.from_str(obj.get("transaction_id"))
         amount = parsers.from_int(obj.get("amount"))
         memo = parsers.from_str(obj.get("memo"), True)
         payee_id = parsers.from_str(obj.get("payee_id"), True)
-        category_id = parsers.from_str(obj.get("category_id"))
+        category_id = parsers.from_str(obj.get("category_id"), True)
```

Comment:
Please make this field Optional since its nullable

Reply:
hi
Yes, i've started using black to format code